tests/periph_flashpage/main.c

```diff
@@ -196,22 +196,33 @@ static uint32_t getaddr(const char *str)
 
 static int cmd_write_raw(int argc, char **argv)
 {
+#ifdef CPU_MSP430FXYZ
```

Comment:
Can we use

`#if __SIZEOF_POINTER__ == 2`

instead for a more general solution?

Reply:
Thanks @benpicco!, I wanted something like! Addressed